src/main/java/problems/impl/LargeValuesAdderImpl.java

```diff
@@ -0,0 +1,67 @@
+package problems.impl;
+
+import problems.interfaces.LargeValuesAdder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+
+public class LargeValuesAdderImpl implements LargeValuesAdder {
+
+    /**
+     * Returns a list of integers that represents the addition outcome after adding two large values represented by
+     * a list of characters.
+     *
+     * @param largeValues1 a list of characters, which represent a large numerical value
+     * @param largeValues2 a list of characters, which represent a large numerical value
+     * @return list of integers that represents the addition outcome after adding two large values
+     */
+    @Override
+    public List<Integer> largeValuesAdder(final List<Character> largeValues1, final List<Character> largeValues2) {
```

Comment:
Why not receive two `String`s? They are `CharSequence`s. Building a number using a list of `Characters` is very hard and also, unreadable.